packages/tailwindcss/src/plugin-api.ts

```diff
@@ -24,15 +30,23 @@ export type PluginAPI = {
     options?: Partial<{
       type: string | string[]
       supportsNegativeValues: boolean
-      values: Record<string, string>
+      values: { __BARE_VALUE__?: (value: NamedUtilityValue) => string | undefined } & Record<
+        string,
+        string
+      >
       modifiers: 'any' | Record<string, string>
     }>,
   ): void
+  theme(path: string): any
 }
 
 const IS_VALID_UTILITY_NAME = /^[a-z][a-zA-Z0-9/%._-]*$/
 
-export function buildPluginApi(designSystem: DesignSystem, ast: AstNode[]): PluginAPI {
+export function buildPluginApi(
```

Comment:
Not really related to this PR but I _think_ we don't have to export this now that we are exporting `registerPlugins` instead 👍

Reply:
Done.